ports/flashlight-cpu/portfile.cmake

```
@@ -0,0 +1,70 @@
+if (EXISTS "${CURRENT_INSTALLED_DIR}/share/flashlight-cuda")
+  message(FATAL_ERROR "flashlight-cuda is installed; only one Flashlight "
+    "backend package can be installed at once. Uninstall and try again:"
+    "\n    vcpkg remove flashlight-cuda\n")
+endif()
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO facebookresearch/flashlight
+    REF 0948e97dbff23d474500a5f55bcf59d3b3589cf4
+    SHA512 89c2efdebc6688a38f9fc3ebb8442c8fe2ad3d51c0e60c5dd8e7ca283715e07f48083543f09eed0db9929d86b765a898d9b3960a9b6f9844b20260f94cf71fdf
+    HEAD_REF master
+)
+
+################################### Build ###################################
+# Default flags
+set(FL_DEFAULT_VCPKG_CMAKE_FLAGS
+  -DFL_BUILD_TESTS=OFF
+  -DFL_BUILD_EXAMPLES=OFF
+  -DFL_BACKEND=CPU # this port is CPU-backend only
+  -DFL_BUILD_STANDALONE=OFF
+  -DFL_INSTALL_CMAKE_DIR=${CURRENT_PACKAGES_DIR}/share/${PORT} # for CMake configs/targets
+)
+
+# Determine which components to build via specified feature
+vcpkg_check_features(
+  OUT_FEATURE_OPTIONS FL_FEATURE_OPTIONS
```

```suggestion
  OUT_FEATURE_OPTIONS FEATURE_OPTIONS
```
Please use the default name.